decidim-core/config/locales/en.yml

```diff
@@ -1358,6 +1358,8 @@ en:
         conversations: Conversations
         followers: Followers
         following: Follows
+        group_admins: Group admins
+        group_members: Group members
```

Comment:
Lets change these strings to "Manage admins" and "Manage members" 
```suggestion
        group_admins: Manage admins
        group_members: Manage members
```

As that's what the frontend is showing:

![Screenshot of the "Manage members" page](https://github.com/decidim/decidim/assets/717367/89ee1854-c059-4a39-8712-e286d517fa02)

The URL is http://localhost:3000/profiles/donnie/group_members (where `donnie` is the group nickname)